src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/AuthorisedServicesIntegrationTest.java

```diff
@@ -0,0 +1,99 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
```

Comment:
why are these java files not in master already? where r u merging this from?

Reply:
Synced master branch